src/libAtomVM/opcodesswitch.h

```diff
@@ -6863,7 +6887,7 @@ HOT_FUNC int scheduler_entry_point(GlobalContext *glb)
                                     TRACE("bs_match/3: error extracting integer.\n");
                                     goto bs_match_jump_to_fail;
                                 }
-                                term t = term_make_maybe_boxed_int64(ctx, value.s);
+                                term t = term_make_maybe_boxed_int64(value.s, &ctx->heap);
```

Comment:
Let's ensure memory also here, since it has bee removed from `term_make_maybe_boxed_int64`.